molecule/default/tests/test_cinder_service.py

```diff
@@ -4,17 +4,18 @@
 import testinfra.utils.ansible_runner
 
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
-    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('utility_container')
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('nova_compute')[:1]
 
 
 @pytest.mark.jira('asc-222')
 def test_cinder_service(host):
-    """Test to verify that cinder service is running on the cinder nodes
-
-    Args:
-        host(testinfra.host.Host): A hostname in dynamic_inventory.json/molecule.yml
-    """
+    """Test to verify that cinder service is running
 
+        Args:
+            host(testinfra.host.Host): A hostname in dynamic_inventory.json/molecule.yml
+        """
```

Comment:
Is the indentation of the previous 3 lines correct?

Reply:
It looks wrong, weird that `pycodestyle` didn't complain. I'll update it quickly